azure-pipelines.yaml

```diff
@@ -103,6 +109,42 @@ steps:
     qualifyImageNames: true
     additionalImageTags: $(containerTag)
   displayName: Push container to registry
+  # build migration image
+- task: DockerCompose@0
```

Comment:
Just an observation, currently the migration image is identical to the release image.
Thie migration image is required, the issue is in the release image which should be built without dev dependencies.